test/rendering-test/node-rendering.js

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2015 - 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+// Enables ES2015 import/export in Node.js
+
+const {execFile} = require('child_process');
+const puppeteer = require('puppeteer');
+const console = require('console');
+const process = require('process');
+
+async function getColor(page, selector) {
+  await page.waitForSelector(selector);
+  const color = await page.evaluate(sel => {
+    return document.querySelector(sel).style.color; //eslint-disable-line
+  }, selector);
+  return color;
+}
+
+async function validateRendering(child) {
+  const browser = await puppeteer.launch({
+    headless: false,
+    executablePath: '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
```

Comment:
Do we really need to specify the path here? Shouldn't it use it by default?

Reply:
By default, it will launch chromium which doesn't work well in rendering.